frontend/packages/noobaa-storage-plugin/integration-tests/tests/objectServiceDashboard.scenario.ts

```diff
@@ -0,0 +1,214 @@
+  });
+
+  it('Create an Object Bucket Claim and test equality', async () => {
+    await browser.wait(until.visibilityOf(obcCount));
+    let text = await obcCount.getText();
+    const [initialCount] = text.split(' ');
+    execSync(`echo '${JSON.stringify(testBucket)}' | kubectl create -f -`);
+    await waitFor(obcCount, (Number(initialCount) + 1).toString());
+    text = await obcCount.getText();
+    const [finalCount] = text.split(' ');
+    expect(Number(finalCount)).toEqual(Number(initialCount) + 1);
+    execSync(`echo '${JSON.stringify(testBucket)}' | kubectl delete -f -`);
+  });
+});
+
+if (!isRGWPresent) {
+  describe('Test Status Card', () => {
+    beforeAll(async () => {
+      await loadAndWait();
+    });
+
+    it('Check if Multi Cloud Gateway is in a healthy state', async () => {
+      await browser.wait(until.and(untilNoLoadersPresent));
+      await browser.wait(until.visibilityOf(healthOfMCG));
+      const color = await healthOfMCG.getAttribute('fill');
+      expect(color).toBe(OCP_HEALTH_ICON_COLORS.GREEN);
+    });
+
+    it('Check if Data Resiliency of MCG is in healthy state', async () => {
+      await browser.wait(until.visibilityOf(resiliencyOfMCG));
+      const color = await resiliencyOfMCG.getAttribute('fill');
+      expect(color).toBe(OCP_HEALTH_ICON_COLORS.GREEN);
+    });
+  });
+}
+
+describe('Test Object Data Reduction Card', () => {
+  beforeAll(async () => {
+    await loadAndWait();
+  });
+
+  it('Check if Efficiency Ratio is in acceptable data range', async () => {
+    await browser.wait(until.and(untilNoLoadersPresent));
+    await browser.wait(until.visibilityOf(efficiencyValue));
+    const effValue = await efficiencyValue.getText();
+    const [ratioA, ratioB] = effValue.split(':');
+    const [numA, numB] = [Number(ratioA), Number(ratioB)];
+    expect(numA).toBeGreaterThan(0);
+    expect(numB).toBeGreaterThan(0);
+  });
+
+  it('Check for savings value to be in acceptable data range', async () => {
+    await browser.wait(until.visibilityOf(savingsValue));
+    const savVal = await savingsValue.getText();
+    const [savDigits] = savVal.split(' ');
+    const numSav = Number(savDigits);
+    if (Number.isNaN(numSav)) {
+      expect(savVal.trim()).toEqual('No Savings');
+    } else {
+      expect(numSav).toBeGreaterThanOrEqual(0);
+    }
+  });
+});
+
+describe('Test Resource Providers card', () => {
+  beforeAll(async () => {
+    await loadAndWait();
+  });
+
+  it('Check if resource provider has at least 1 provider', async () => {
+    await browser.wait(until.and(untilNoLoadersPresent));
+    await browser.wait(until.visibilityOf(resourceProviders));
+    const text = await resourceProviders.getText();
+    expect(text).toBeDefined();
+  });
+});
+
+if (isRGWPresent) {
+  describe('Test RGW Dashboard items are shown', () => {
+    beforeAll(async () => {
+      await loadAndWait();
+      await browser.sleep(10 * SECOND);
+    });
+
+    if (isRGWPresent) {
```

Comment:
Why do you need to check for RGW presence again here as you have checked it at line 126?

Reply:
good catch